Makefile

```diff
@@ -76,7 +76,7 @@ dev-depext:
 
 .PHONY: melange
 melange:
-	opam pin add -n melange.dev https://github.com/melange-re/melange.git#v4-414-dev
+	opam pin add -n melange.dev https://github.com/melange-re/melange.git#24e21cc42
```

Comment:
Is it worth adding a comment to explain to which branch the hash is pointing to? Like in the nix case. For those of us that are not hip 😄